agent-manager/tests/test_agent_manager.py

```diff
@@ -0,0 +1,18 @@
+import pytest
```

Comment:
I'm thinking for now the tests directory should likely be under the local testing directory as well?